src/components/OpportunityCard/OpportunityCard.tsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+
+type CardType = 'work' | 'education';
```

Comment:
Of course this could already be the actual values when deciced, if something along lines of `'työ' | 'kurssi'`